.idea/gradle.xml

```diff
@@ -5,7 +5,7 @@
       <GradleProjectSettings>
         <option name="distributionType" value="DEFAULT_WRAPPED" />
         <option name="externalProjectPath" value="$PROJECT_DIR$" />
-        <option name="gradleJvm" value="1.8" />
```

Comment:
Okay. But check your Java version. We should make sure of we're using the same (to minimize compatibility issues?)